software/v2/include/state_machine.hpp

```diff
@@ -5,26 +5,23 @@
 // Event Declarations
 
 struct HEALTH_fail : tinyfsm::Event { };
-struct LOCK_cmd : tinyfsm::Event { };
-struct ARM_cmd : tinyfsm::Event { };
 struct LAUNCH_cmd : tinyfsm::Event{ };
-struct ESTOP_cmd : tinyfsm::Event{ };
+struct ON_cmd : tinyfsm::Event{ };
+struct BRAKE_cmd : tinyfsm::Event{ };
 struct END_OF_TRACK : tinyfsm::Event{ };
```

Comment:
Isn't "END_OF_TRACK" redundant with BRAKE_cmd ? While driving, you only brake when you're approaching the end of the track right ?